pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -597,6 +597,17 @@ func parseAppInstanceConfig(config *zconfig.EdgeDevConfig,
 		appInstance.FixedResources.VncPasswd = cfgApp.Fixedresources.VncPasswd
 		appInstance.FixedResources.DisableLogs = cfgApp.Fixedresources.DisableLogs
 		appInstance.MetaDataType = types.MetaDataType(cfgApp.MetaDataType)
+		// Calculate the moment when the application should start, taking into account the configured delay
+		startTime := getconfigCtx.zedagentCtx.delayBaseTime.Add(time.Duration(cfgApp.StartDelayInSeconds) * time.Second)
+		appInstance.StartTime = startTime
+		// Is it the first config that we process? In this case, the apps that come with this config - are predefined
+		// and should be delayed
```

Comment:
Why does this matter?

Reply:
It's the code from the extra commit that tries to handle the requirement "we do not apply the delay for the apps started after the system is up and running".
I've tried to explain it in the commit message.

Reply:
It's the code from the extra commit that tries to handle the requirement "we do not apply the delay for the apps started after the system is up and running".

Reply:
Mmm... Have we discussed it with Gopi? 
I have an impression that he still insists on this feature.